File: app/src/main/java/com/rodolfonavalon/canadatransit/model/transit/Response/OperatorsResponse.java
package com.rodolfonavalon.canadatransit.model.transit.Response;

import com.google.gson.annotations.SerializedName;
import com.rodolfonavalon.canadatransit.model.database.Operator;

import java.util.List;

import lombok.Getter;

public class OperatorsResponse extends MetaResponse {

    @SerializedName("operators")
    @Getter private List<Operator> operators;
}

File: app/src/main/kotlin/com/rodolfonavalon/canadatransit/model/transit/response/OperatorsResponse.kt
package com.rodolfonavalon.canadatransit.model.transit.response

import com.rodolfonavalon.canadatransit.model.database.transit.Operator
import com.rodolfonavalon.canadatransit.model.transit.Meta
import com.squareup.moshi.Json

class OperatorsResponse(
    @field:Json(name = "operators") override val response: List<Operator>,
    @field:Json(name = "meta") override val meta: Meta
) : MetaResponse<Operator>()
